packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -389,4 +388,14 @@ module.exports = {
       }
     }
   },
-}
+  async ready() {
+    if (!process.env.PROD_ENV) {
+      ctx.pages.forEach((page) => {
+        if (!page.frontmatter['meta']) {
+          page.frontmatter['meta'] = []
+        }
+        page.frontmatter['meta'].push({ name: 'robots', content:'noindex,nofollow'});
```

Comment:
Add a comment here to mention why we are adding the meta tag

Reply:
done

Reply:
I can understand adding **noindex** to block the search engines from indexing the page but why are we adding **nofollow**?  **nofollow** is used when the search engine crawls through other links on your page.

Reply:
Since this is only for the preview trex site, we might have links to pages that are also not public yet, so it's reasonable to say `nofollow` as well here.